readme.md

```diff
@@ -1,151 +1,131 @@
 
-> [!NOTE]
-> This extension is created and maintained personally by a Renovate developer/user so support requests relating to the extension itself cannot be answered directly in the main Renovate repository.
+### Comparisons
 
-#### Custom pipeline
+- [Different ways to run Renovate](https://www.mend.io/renovate/)
+- [Renovate vs Dependabot Comparison](https://docs.renovatebot.com/bot-comparison/)
 
-You can create a custom pipeline with a `yml` definition that triggers `npx renovate`.
-[More details on how to configure the pipeline](https://docs.renovatebot.com/modules/platform/azure/).
+## Get involved
 
-### Bitbucket Cloud/Server, Forgejo, Gitea, GitLab
+### Issues and Discussions
 
-For Bitbucket Cloud, Bitbucket Server, Forgejo, Gitea and GitLab, use our self-hosting option.
+We prefer all items to be opened as a discussion before opening an issue.
 
-## Configuration
+- [GitHub Discussions forum](https://github.com/renovatebot/renovate/discussions)
 
-Go to our [documentation website](https://docs.renovatebot.com/) to learn how to configure Renovate.
-We have a [full list of configuration options](https://docs.renovatebot.com/configuration-options/).
+### Contributing
 
-To get help with your configuration, go to the [discussions tab in the Renovate repository](https://github.com/renovatebot/renovate/discussions) and open a new "config help" discussion post.
+To contribute to Renovate, or run a local copy, please read the contributing guidelines.
 
-## Self-Hosting
+- [Guidelines for Contributing](https://github.com/renovatebot/renovate/blob/main/.github/contributing.md)
+- Items that need contribution: [good first issues](https://github.com/renovatebot/renovate/contribute)
 
-To run your own instance of Renovate you have several options:
+### Contact and Social Media
 
-- Install the `renovate` CLI tool from npmjs, run it on a schedule (e.g. using `cron`)
-- Run the `renovate/renovate:full` Docker Hub image (same content/versions as the CLI tool), run it on a schedule
-- Run the `renovate/renovate:latest` Docker Hub image if you only use package managers that don't need third-party binaries (e.g. JavaScript, Docker, NuGet, pip)
+The Renovate project is proudly supported and actively maintained by Mend.io.
 
-[More details on the self-hosting development](https://github.com/renovatebot/renovate/blob/main/docs/usage/getting-started/running.md).
+- Contact [Mend.io](http://Mend.io) for commercial support questions.
 
-## Contributing
+Follow us on Twitter, LinkedIn and Mastodon.
 
-If you want to contribute to Renovate or get a local copy running, please read the instructions in [contributing guidelines](.github/contributing.md).
-To get started look at the [list of good first issues](https://github.com/renovatebot/renovate/contribute).
+- Twitter: [x.com/mend_io](http://x.com/mend_io)
+- LinkedIn: [linkedin.com/company/mend-io](https://linkedin.com/company/mend-io)
```

Comment:
The list is missing the Mastodon contact info. :wink:

Reply:
Removed the detail from the list heading per your suggestion. Thanks